_episodes/08-software.md

```diff
@@ -0,0 +1,14 @@
+# Applying FAIR to software
```

Comment:
You'll need  to add some metadata to the front of the file here so that it has a title and appears in the lesson navigation. You should be able to copy and amend the info from one of the other episodes: take the bit between and including the three dashes:

```yaml
---
title: "Whatever"
# ... some other metadata that you also need
---
```

Reply:
I added some metadata, not sure if I did the commit/pull request right. Do you see the latest update?

Reply:
I think you've done the pull request correctly - I'm going to approve this so I can make some suggestions to the metadata for the lesson navigation box.